src/database/seeders/20190708175931-article.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  up: (queryInterface /* ,  Sequelize */) =>
+    queryInterface.bulkInsert('Articles', [
+      {
+        // id: 1,
+        title: 'Sermon',
+        description: 'The word',
+        content:
+          'In the beginning was the word, the word was with us and the word was God',
+        image: 'image1',
+        slug: 'article1',
+        status: 'draft',
+        // published_date: new Date(),
+        // authorId: 1,
+        created_at: new Date(),
+        updated_at: new Date()
+      },
+      {
+        // id: 2,
+        title: 'Sermon ====>',
+        description: 'The word  of',
+        content:
+          'In the beginning was the word, the word was with us and the word was God',
+        image: 'image1',
+        slug: 'article1',
+        status: 'draft',
+        // published_date: new Date(),
+        // authorId: 1,
+        created_at: new Date(),
+        updated_at: new Date()
+      }
+    ]),
+
+  down: (queryInterface /* ,  Sequelize */) => {
+    return queryInterface.bulkDelete('Articles', null, {});
+  }
+};
```

Comment:
Please remove these lines you commented out if you are not using them.

Reply:
Thanks for this review.
DONE